paper.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[journal,a4paper]{IEEEtran}
+\documentclass[journal,10pt,a4paper]{IEEEtran}
```

Comment:
Is it consistent to the template specific to IEEE ECICE 2019?

Reply:
No, the template of ECICE has different font size to IEEE tran.

Reply:
I just hesitate with the formation. We should follow the requirements of ECICE 2019.

Reply:
Ok, I'll try another version without the IEEE trans template.
What are we going to submit, the pdf or latex file?

Reply:
We would submit PDF only.

Reply:
At the moment, we don't really need to tweak aggressively.